src/packit_deploy/packit_constellation.py

```diff
@@ -118,6 +119,13 @@ def get_env(cfg):
         "PACKIT_OUTPACK_SERVER_URL": f"http://{cfg.container_prefix}-{outpack}:8000",
         "PACKIT_AUTH_ENABLED": "true" if cfg.packit_auth_enabled else "false",
     }
+    if cfg.branding_enabled:
+        env.update({"PACKIT_BRAND_LOGO_NAME": cfg.brand_logo_name})
+        env.update({"PACKIT_BRAND_LOGO_ALT_TEXT": cfg.brand_logo_alt_text})
+        try:
+            env.update({"PACKIT_BRAND_LOGO_LINK": cfg.brand_logo_link})
+        except AttributeError:
+            print("[packit-api] A brand logo link was not provided (optional)")
```

Comment:
Why not just check if the attribute is set and not attempt to set the value if it isn't provided?